examples/ts/phemex-create-order-position-with-takeprofit-stoploss.ts

```diff
@@ -30,12 +30,12 @@ async function example () {
         'posSide': 'Long', // "Long" / "Short" for hedge mode
         'stopLoss': {
             'triggerPrice': stop_loss_trigger_price,
-            'type': 'limit',
+            'triggerPriceType': 'ByLastPrice',
```

Comment:
this was a typo probably. definitely the `type` param does not do anything in implementation.
we only have `triggerPriceType`

Reply:
@ttodua I think we're talking about difference things, shouldn't we support `type = limit/market` when the exchange allows it? I know we can also infer it by checking the existence of `price` but `type` should also be supported I think

Reply:
but where `type` param is either being used in implementation of phemex createOrder? or where does phemex api docs mention support of it?

Reply:
> but where `type` param is either being used in implementation of phemex createOrder? or where does phemex api docs mention support of it?

@ttodua I think it's like "if type = limit" then we have to extract price from `stopLoss/takeProfit` otherwise we don't and we can use `triggerPrice` only

Reply:
> to extract price from stopLoss/takeProfit

what does that mean / how that works? I think I am not well following what you mean :(

Reply:
what I think is that we are adding unnecessary complexity and non-unified approaches, which I don't think is beneficial.

Reply:
i've restored the `type`, i think this PR now does not contain any glitch